### PR TITLE
vpcv1

### PR DESCRIPTION
**Pull request by Polina-Gubina**

### CODE REVIEW

File: otcextensions/sdk/vpcv1/v1/bandwidth.py [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+from openstack import resource
+from openstack import utils
+
+
```
Comment (vladimirhasko) [inline otcextensions/sdk/vpcv1/v1/bandwidth.py:15]: something like the following is missing:

```
from openstack import proxy


class Proxy(proxy.Proxy):

    skip_discovery = True
```

**Review by vladimirhasko — CHANGES_REQUESTED** [review]

File: otcextensions/sdk/vpcv1/vpc_service.py [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+import warnings
+from urllib.parse import urlparse
+
+from openstack import exceptions
+from openstack import service_description
+
+from otcextensions.sdk.vpc.v1 import _proxy
```
Comment (vladimirhasko) [inline otcextensions/sdk/vpcv1/vpc_service.py]: from otcextensions.sdk.vpcv1.v1 import _proxy

**Review by vladimirhasko — CHANGES_REQUESTED** [review]

**Comment by artem-lifshits** [comment]
Recheck

**Comment by vladimirhasko** [comment]
recheck

**Comment by vladimirhasko** [comment]
recheck

**Pull request closed without merging**